app/services/sources/properties-managers/streamlabels-manager.ts

```diff
@@ -14,7 +14,7 @@ export class StreamlabelsManager extends PropertiesManager {
   @Inject() userService: UserService;
 
   settings: IStreamlabelsManagerSettings;
-  filename: string;
+  subscription: IStreamlabelSubscription;
```

Comment:
For easier refactoring it's better to use `private` modifier by default for each parameter that we are not going to use in vue components and outside of this service. When you will try to remove or rename private property, IDE will try to find usages only in current file instead of whole project.